etc/config/cuda.amazon.properties

```diff
@@ -144,16 +144,30 @@ compiler.cltrunk.exe=/opt/compiler-explorer/clang-trunk/bin/clang++
 compiler.cltrunk.demangler=/opt/compiler-explorer/gcc-snapshot/bin/c++filt
 compiler.cltrunk.options=--gcc-toolchain=/opt/compiler-explorer/gcc-snapshot --cuda-path=/opt/compiler-explorer/cuda/11.3.1 --cuda-gpu-arch=sm_86 --cuda-device-only -Wno-unknown-cuda-version
 
-group.hipclang.compilers=hiptrunk
+group.hipclang.compilers=hipclang-rocm-40502:hipclang-rocm-50002:hipclang-rocm-50103:hipclang-rocm-50203
 group.hipclang.isSemVer=true
-group.hipclang.baseName=clang (HIP)
+group.hipclang.baseName=clang
 group.hipclang.compilerType=clang-hip
 group.hipclang.objdumperType=llvm
-group.hipclang.options=-x hip --offload-arch=gfx908 --cuda-device-only -nocudainc -nocudalib -include __clang_hip_runtime_wrapper.h
 
-compiler.hiptrunk.semver=trunk
```

Comment:
can you leave hiptrunk in here, but mark it as `compiler.hiptrunk.hidden=true`

Reply:
Sure. I've added hiptrunk back with all its original settings and just marked it as hidden.